packages/graph/hash_graph/lib/graph/src/store/postgres/query/compile.rs

```diff
@@ -46,11 +46,11 @@ impl<'f: 'q, 'q, T: PostgresQueryRecord<'q>> SelectCompiler<'f, 'q, T> {
     }
 
     /// Creates a new compiler, which will default to select the fields returned from
```

Comment:
Would we still call these fields here?

Reply:
Actually, I think I will remove this method entirely later on, but I have not decided yet. This implies that requesting from a row is closer to the specified selection.

Reply:
Could you say more about what "requesting from a row is closer to the specified selection" means?

Reply:
#1258 currently uses this method to create a compiler with the default selection. A few lines after creating the compiler it assumes that the returned `Row` has some specific columns.

With this PR it's possible to always use the default compiler and manually adding the `Path`, which then can be returned by `row.get(x)`.